package.json

```diff
@@ -68,7 +68,7 @@
     "file-loader": "^0.8.5",
     "identity-obj-proxy": "^3.0.0",
     "imports-loader": "^0.6.5",
-    "jest-cli": "^16.0.1",
+    "jest-cli": "19.1.0-alpha.eed82034",
```

Comment:
I'd second that it's fine, mostly because it's a dev dependency. Let's keep an eye out to update soon.